core/src/main/java/io/grpc/Status.java

```diff
@@ -377,8 +376,7 @@ public static Status fromCode(Code code) {
   /**
    * Key to bind status message to trailing metadata.
    */
-  @Internal
-  public static final Metadata.Key<String> MESSAGE_KEY =
+  static final Metadata.Key<String> MESSAGE_KEY =
```

Comment:
I tried it that way, but it ended up being ugly.  There are parts of Status.Code which need to be exposed too.  It pulled too much of Status and Code into internal, rather than just the internal bits.